monkey/tests/data_for_tests/agent_plugin/src/requirements.txt

```diff
@@ -0,0 +1 @@
+bcrypt
```

Comment:
Is bcrypt a good choice for this? It's written in Rust, which means this dependency is not cross platform.

We need something simpler.